jetty-http/src/test/java/org/eclipse/jetty/http/HttpParserTest.java

```diff
@@ -112,15 +120,15 @@ public void testLineParse1_RFC2616() throws Exception
         ByteBuffer buffer = BufferUtil.toBuffer("GET /999\r\n");
 
         HttpParser.RequestHandler handler = new Handler();
-        HttpParser parser = new HttpParser(handler, HttpCompliance.RFC2616);
+        HttpParser parser = new HttpParser(handler, HttpCompliance.RFC2616_LEGACY);
```

Comment:
This is probably not the best place for this comment, but  ....

So, IIUC, folks who were using HttpCompliance.RFC2616 now have to use HttpCompliance.RFC2616_LEGACY to get the same behaviour???  If so, I think I prefer the other alternative of RFC2616_STRICT for a strict adherence to the spec, so no user code changes are required.

Reply:
@janbartel has a point here with backward compatibility with existing users.

+1 for _STRICT

Reply:
I disagree.  It is a bug that our RFC2616 compliance mode was not actually RFC2616 compliant.  Users that select that mode want compliance with the RFC and not with a legacy interpretation of it by jetty.  As we find/fix bugs with our RFC compliance, we need to be able to update the RFC compliance modes.  Users that wish to be compatible with bugs can either edit the compliance modes or they can use a legacy mode.